src/__tests__/patients/appointments/AppointmentsList.test.tsx

```diff
@@ -45,26 +45,35 @@ const history = createMemoryHistory()
 
 let store: any
 
-const setup = (patient = expectedPatient, appointments = expectedAppointments) => {
+const setup = async (patient = expectedPatient, appointments = expectedAppointments) => {
   jest.resetAllMocks()
   jest.spyOn(PatientRepository, 'getAppointments').mockResolvedValue(appointments)
   store = mockStore({ patient, appointments: { appointments } } as any)
-  const wrapper = mount(
-    <Router history={history}>
-      <Provider store={store}>
-        <AppointmentsList patientId={patient.id} />
-      </Provider>
-    </Router>,
-  )
+
+  let wrapper: any
+
+  await act(async () => {
+    wrapper = await mount(
+      <Router history={history}>
+        <Provider store={store}>
+          <AppointmentsList patientId={patient.id} />
+        </Provider>
+      </Router>,
+    )
+  })
+
+  wrapper.update()
+
   return wrapper
```

Comment:
small refactor: let's change this to be `return { wrapper: wrapper as ReactWrapper }`. This makes the code more extensible in the future and gives type support for wrapper.